homeassistant/components/calendar/trigger.py

```diff
@@ -174,9 +187,11 @@ async def async_attach_trigger(
         "platform": DOMAIN,
         "event": event_type,
     }
+    if offset:
```

Comment:
Maybe always include the offset to make it more predictable for users?

Reply:
👍🏼 

I noticed the sun and calendar offset strings can be different format than the input strings, and they look difficult to work with. Is there a common template function users use to get something useful out of the offset? e.g it can be something like "-1day 15 minutes" that seem hard to work with without a helper function.